Acornfile

```diff
@@ -27,15 +32,15 @@ containers: {
 			ACORN_DB_ENGINE: "mariadb"
 			ACORN_DB_USER: "secret://db-user-credentials/username?onchange=no-action"
 			ACORN_DB_PASSWORD: "secret://db-user-credentials/password?onchange=no-action"
-			ACORN_DB_HOST: "db"
+			ACORN_DB_HOST: args.dbHost
 			ACORN_DB_PORT: "3306"
 			ACORN_DB_NAME: args.dbName
 			ACORN_ROUTE53_ZONE_ID: args.route53ZoneId
 			AWS_ACCESS_KEY_ID: "secret://aws-creds/access-key?onchange=no-action"
 			AWS_SECRET_ACCESS_KEY: "secret://aws-creds/secret-key?onchange=no-action"
 		}
 	},
-
+    if args.dbHost == "db" {
```

Comment:
probably 6 of one half dozen of the other, but I'd do `!= local`. Leave it to you.

Reply:
I'm going to leave this as it is for now, and refactor later as we adopt the profiles/services stuff.